bigframes/session/__init__.py

```diff
@@ -1456,15 +1456,37 @@ def read_gbq_function(
             The return type of the function must be explicitly specified in the
             function's original definition even if not otherwise required.
 
+        BigQuery UDFs has many public functions under the ``bqutil`` project on publicly shared datasets
+        (See: https://github.com/GoogleCloudPlatform/bigquery-utils/tree/master/udfs#using-the-udfs).
+        You can checkout Community UDFs to use community-contributed functions.
+        (See: https://github.com/GoogleCloudPlatform/bigquery-utils/tree/master/udfs/community#community-udfs).
+
         **Examples:**
 
+        Using the ``cw_lower_case_ascii_only`` function from Community UDFs.
```

Comment:
Nit: verb should be active, telling the user what to do.

```suggestion
        Use the ``cw_lower_case_ascii_only`` function from Community UDFs.
```